### PR TITLE
XR docs

### PR DESCRIPTION
**Pull request by Sanakan8472**

### CODE REVIEW

File: pages/docs/xr/xr-project-setup.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# XR Project Setup
+
+## XR Project Config
+
+As an example, we will use the [Testing Chambers](../../samples/testing-chambers.md) project and its **Surfaces** scene to set up XR rendering for desktop VR use. Start by opening the project and scene.
+
+Enable *Show in Development Features* in the [editor settings](..\editor\editor-settings.md) and restart it to see XR features in the editor.
+
+To use XR in your project you must first load a plugin to supports XR devices. In the [plugin selection](../projects/plugin-selection.md) dialog, select your custom XR plugin. In this case, we select the *Open XR* plugin and close the dialog.
```
Comment (jankrassnigg) [inline pages/docs/xr/xr-project-setup.md:9]: support without S

File: pages/docs/xr/xr-project-setup.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+* `Device Type`: Which device we want to track. E.g. the HMD (a.k.a. your head) or one of the controllers.
+* `Pose Location`: Some input devices have different poses for `Grip` and `Aim`. E.g. your hand's grip position is in the middle of your fist pointing upwards, while the aim position is at your index finger pointing forward.
+* `Transform Space`: Whether the local or global transform should be set to the input device's transform.
+* `Rotation`: Whether to apply rotation to the game object. Translation is always applied but in some cases it might be useful to not apply rotation. E.g. for helper nodes in the scene.
+* `Scale`: Whether to apply scale to the game object. All input devices usually have an identity scale. If you don't want this to be overwritten while tracking, disable this option.
+
+Add one component to the `Camera` game object with default settings and one to the `Gun` game object with device type `Right Controller`, pose location `Aim` and `Global` transform.
+
+Safe the prefab and play the scene again. You should now be able to move the controller via a gamepad and poin the gun at things using your right controller.
```
Comment (jankrassnigg) [inline pages/docs/xr/xr-project-setup.md:40]: poin -> point

**Review by jankrassnigg — APPROVED** [review]

File: pages/docs/xr/xr-project-setup.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## See Also
+
+* [XR Graphics](xr-graphics.md)
+* [XR Input](xr-input.md)
```
Comment (jankrassnigg) [inline pages/docs/xr/xr-project-setup.md:45]: These pages don't exist, right ?

Comment (Sanakan8472) [reply pages/docs/xr/xr-project-setup.md:45]: Not yet, that's why it's a draft. these will follow shortly.

Comment (Sanakan8472) [reply pages/docs/xr/xr-project-setup.md:40]: Added now! PR is no longer a draft. Everything should be present now.

File: pages/docs/xr/xr-components.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+## Device Tracking Component
+
+To automatically make a game object follow the HMD or one of the controllers, you can attach a `ezDeviceTrackingComponent` to a game object.
+
+### Component Properties
+* `Device Type`: Which device we want to track. E.g. the HMD (a.k.a. your head) or one of the controllers.
+* `Pose Location`: Some input devices have different poses for `Grip` and `Aim`. E.g. your hand's grip position is in the middle of your fist pointing upwards, while the aim position is at your index finger pointing forward.
+* `Transform Space`: Whether the local or global transform should be set to the input device's transform.
+* `Rotation`: Whether to apply rotation to the game object. Translation is always applied but in some cases it might be useful to not apply rotation. E.g. for helper nodes in the scene.
+* `Scale`: Whether to apply scale to the game object. All input devices usually have an identity scale. If you don't want this to be overwritten while tracking, disable this option.
```
[the remark this answers is not in the document]
Comment (Sanakan8472) [reply pages/docs/xr/xr-components.md]: I've changed the description. It makes sense if the `transform space` is set to `global` checking the box will make the object match the scale of the stage space.

**Pull request merged**